econet_base.yaml

```diff
@@ -40,6 +40,12 @@ api:
         datapoint_id: string
       then:
         - lambda: id(econet_id).homeassistant_read(datapoint_id);
+    - service: read_from_address
```

Comment:
Do we really need a new service? Can't you have the address as optional?

Reply:
Answered in:
https://github.com/esphome-econet/esphome-econet/pull/250#discussion_r1423102535